tests/store/tracking/test_sqlalchemy_store_query_trace_metrics.py

```diff
@@ -754,3 +755,176 @@ def test_query_trace_metrics_with_invalid_filter(
             aggregations=[MetricAggregation(aggregation_type=AggregationType.COUNT)],
             filters=[filter_string],
         )
+
+
```

Comment:
Will the interpolated results from postgres and the estimates from other DB engines produce odd artfiacts for percentile values if an array of identical values are presented? If there are floating point rounding issues in this scenario, can we confirm that we don't get bogus rounding errors like `1e-16`?

Reply:
Yes makes sense, we may want to round it by default. Will add this in a small follow up PR